src/frontend/src/lib/types/coingecko.ts

```diff
@@ -6,7 +6,7 @@ export type CoingeckoCoinsId = 'ethereum' | 'bitcoin' | 'internet-computer';
 
 // We are interested in the ERC20 <> USD on Ethereum only, therefore not an exhaustive list.
```

Comment:
The comments does not match anymore

Reply:
i had updated it, I think i forgot to push the commit